test/extensions/bootstrap/wasm/config_test.cc

```diff
@@ -84,15 +84,16 @@ TEST_P(WasmFactoryTest, CreateWasmFromWasmPerThread) {
 TEST_P(WasmFactoryTest, MissingImport) {
 
   envoy::extensions::wasm::v3::WasmService config;
+  config.mutable_config()->set_name("test");
   config.mutable_config()->mutable_vm_config()->set_runtime(
       absl::StrCat("envoy.wasm.runtime.", GetParam()));
   config.mutable_config()->mutable_vm_config()->mutable_code()->mutable_local()->set_filename(
       TestEnvironment::substitute(
           "{{ test_rundir }}/test/extensions/bootstrap/wasm/test_data/missing_cpp.wasm"));
   config.set_singleton(true);
 
-  EXPECT_THROW_WITH_REGEX(initializeWithConfig(config), Extensions::Common::Wasm::WasmException,
-                          "Failed to load Wasm module due to a missing import: env.missing");
+  EXPECT_THROW_WITH_MESSAGE(initializeWithConfig(config), Extensions::Common::Wasm::WasmException,
+                            "Unable to create Wasm service test");
```

Comment:
This doesn't test what it is supposed to test anymore. What happened to the original error message? The cpp-host part still emits it.

Reply:
The original error message is coming out in a log statement.  Is there any way of capturing those log statements in tests without having to resort to intrusive changes e.g. adding code to intercept the logs manually ?  Alternatively I can stored the original log message as say fail_message_ and pass it back via the exception.

Reply:
I guess it's fine if it's emitted in the logs. It's a bit unfortunate that we emit 2 separate messages, and normally I'd ask to retain and resurface the root source of failure, but I have no idea what upstream's take on that will be, so let's leave it as-is, and let them bikeshed.